bin/package_ios

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+###########################################################################
+# Packaging script which creates iOS framework (Swift compatible).
+#
+# Usage:
+#> bin/package_ios <architecture>
+#
+# Package (specific architecture: arm64, amd64):
+#> bin/package_ios arm64
+
+set -e
+
+source bin/helpers/output.sh
+source bin/helpers/functions.sh
+
+ARCH=$1
+if [[ ! "$ARCH" =~ ^(arm64|amd64)$ ]]; then
```

Comment:
Is this necessary? AFAIK xcode strips out unnecessary architectures from fat binaries on it's own so I think we should include all supported architectures and allow users of the framework to decide which architectures they'll need. Default `gomobile bind -target=ios` contains `Architectures in the fat file: Mysterium.framework/Mysterium are: armv7 i386 x86_64 arm64`, note that `armv7` is needed to support older devices.

Reply:
I thought on mobile size matters: 
- 24MB binary size with single architecture 
- 100MB binary size with all architectures

Reply:
https://pspdfkit.com/guides/ios/current/troubleshooting/removing-architectures/ both `cocoapods` and `carthage` automatically remove desktop architectures from release builds. 
In our case desktop architectures are not important for ios because we're using the framework in the network extension which can only run on real devices, but we still need both `armv7` and `arm64` architectures inside the `.framework` file.

Reply:
why we need arm7 anyway? I don't even crosscompiled necessary libs for this arch. We are not planning to support pre ios 10.x devices anyway.

Reply:
I used this compability matrix http://iossupportmatrix.com/ and assumed that we start supporting ARMv8 and upwards which is basically ios 11 and greater. So far it will be enough. For older arms support it's a question for POs - what segments to target. The same is for android - initially 64 bit archs. Older ones - later, if needed.

Reply:
in that case yeah, we can just compile `arm64` for ios